content/docs/hooks-overview.md

```diff
@@ -1,107 +1,107 @@
+#### Tetapi Apa itu Hook? {#but-what-is-a-hook}
 
-Hooks are functions that let you “hook into” React state and lifecycle features from function components. Hooks don't work inside classes -- they let you use React without classes. (We [don't recommend](/docs/hooks-intro.html#gradual-adoption-strategy) rewriting your existing components overnight but you can start using Hooks in the new ones if you'd like.)
+*Hooks* merupakan fungsi yang memungkinkan untuk "mengait" pada fitur-fitur *state* dan siklus React dari komponen fungsi. *Hooks* tidak dapat berfungsi didalam kelas -- *Hooks* memungkinkan menggunakan React tanpa kelas. (Kami [tidak merekomendasikan](/docs/hooks-intro.html#gradual-adoption-strategy) membuat ulang komponen yang sudah ada tetapi Anda dapat menggunakan *Hooks* untuk komponen yang baru jika diinginkan.)
 
-React provides a few built-in Hooks like `useState`. You can also create your own Hooks to reuse stateful behavior between different components. We'll look at the built-in Hooks first.
+React menyediakan beberapa *Hooks* bawaan seperti `useState`. Anda dapat membuat *Hooks* sendiri untuk menggunakan ulang *stateful behavior* antara komponen yang berbeda. Kami akan membahas *Hooks* bawaan terlebih dahulu.
 
->Detailed Explanation
+>Penjelasan Detail
 >
->You can learn more about the State Hook on a dedicated page: [Using the State Hook](/docs/hooks-state.html).
+>Anda dapat belajar lebih lanjut mengenai *Hook* State pada laman khusus: [Menggunakan Hook State](/docs/hooks-state.html).
 
-## ⚡️ Effect Hook {#effect-hook}
+## ⚡️ Hook Effect {#effect-hook}
 
-You've likely performed data fetching, subscriptions, or manually changing the DOM from React components before. We call these operations "side effects" (or "effects" for short) because they can affect other components and can't be done during rendering.
+Anda tentunya pernah melakukan pengambilan data, proses berlangganan atau menerima data secara terus menerus (*subscriptions*), atau secara manual merubah DOM dari komponen React sebelumnya. Kami menyebut operasi-operasi tersebut "*side effects*" atau "efek samping" karena dapat mempengaruhi komponen lain dan tidak dapat dilakukan pada saat proses *render*.
 
-The Effect Hook, `useEffect`, adds the ability to perform side effects from a function component. It serves the same purpose as `componentDidMount`, `componentDidUpdate`, and `componentWillUnmount` in React classes, but unified into a single API. (We'll show examples comparing `useEffect` to these methods in [Using the Effect Hook](/docs/hooks-effect.html).)
+Hook Effect, `useEffect`, menambahkan kemampuan untuk melakukan "efek samping" dari sebuah komponen fungsi. Hal tersebut memiliki tujuan yang sama seperti `componentDidMount`, `componentDidUpdate`, dan `componentWillUnmount` pada kelas React, tetapi bersatu menjadi satu API. (Kami akan memberi beberapa contoh perbandingan `useEffect` dengan *method* pada [Menggunakan Hook Effect](/docs/hooks-effect.html).)
```

Comment:
```suggestion
Hook Effect, `useEffect`, menambahkan kemampuan untuk melakukan "efek samping" dari sebuah *function component*. *Hook* ini memiliki fungsi yang sama dengan `componentDidMount`, `componentDidUpdate`, dan `componentWillUnmount` pada kelas React, tetapi disatukan menjadi satu API. (Kami akan memberi beberapa contoh perbandingan `useEffect` dengan metode ini pada [Menggunakan Hook Effect](/docs/hooks-effect.html).)
```

*methods* in this context means `the way things would be done` instead of `programming method` so I think we can translate it.
What do you think? 😄

Reply:
Got it 👍